### PR TITLE
Arbitrary Compliance

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Custom requirements](https://trello.com/c/U52YgpVe/817-custom-requirements)

File: package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "tar": "~0.1.8",
     "tinycolor2": "^1.4.1",
     "u2f": "^0.1.3",
-    "uuid": "^3.1.0",
+    "uuid": "^8.3.2",
```
Comment (josepfo) [inline package.json:45]: Any specific reason for this change?

File: ui/css/main.scss [hunk 2]
```diff
@@ -1516,3 +1530,162 @@ footer {
...
+  border-radius: 10px;
+  background-color: $ligth-color;
+  display: flex;
+  align-items: center;
+  font-family: $font-secondary;
+}
+
+.choice-list-actions-wrapper {
+  position: relative;
+  // width: 100%;
```
Comment (josepfo) [inline ui/css/main.scss]: Can we remove the width?

**Review by josepfo — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (chaotixkilla) [reply package.json:45]: Having access to the `validate()` function

**Review by josepfo — APPROVED** [review]

**Pull request merged**